### DISCUSSION TITLE
成功运行，但没有完成相应任务

### DISCUSSION THREAD

**Discussion by yqlengtong** [0-8-4/miui-auto-tasks#279] [问与答｜Q&A]
运行日志如下，没有报错，但也没有完成任务
<img width="538" alt="{60785656-2CDF-4c9d-A310-B0CA38384A63}" src="https://github.com/0-8-4/miui-auto-tasks/assets/108358875/0f15522c-6441-404b-9d2f-070dc7e696c3">

**Comment by TardisXE** [comment]
检查login_user_agent是否填写，所需功能有没有启用

**Reply by yqlengtong** [reply]
懂了，感谢大佬的解答
